docs/source/index.rst

```diff
@@ -1,7 +1,10 @@
 .. image:: ../Pseudopeople-logo_FINAL_2023.04.11_psdppl-logo_blue-ombre.png
 
 pseudopeople is a Python package that generates realistic simulated data about a
-fictional United States population over multiple decades, for use in testing entity resolution (record linkage)
+fictional United States population that we have simulated over multiple decades.
+This fictional population, which we have generated with a stochastic simulation, is
+sampled and noised to produce datasets for evaluating and comparing data science
+algorithms and systems, such as for use in testing entity resolution
 methods.
```

Comment:
@aflaxman It seems more precise but less snappy than the original version -- some of the wording seems a bit clunky to me. Here's an attempt at making it less clunky and even more precise:

> pseudopeople is a Python package that generates realistic simulated data about a
fictional United States population that has been stochastically simulated over multiple decades. We have sampled this fictional population at various timepoints throughout the simulation, and the pseudopeople package adds noise to the sampled data to produce realistic, noisy datasets for use in testing entity resolution methods or other data science algorithms and systems.

@zmbc Any thoughts?

Reply:
I recognize the necessity to include more information on where the population came from -- I thought users wouldn't care much about that, and I was clearly wrong. But I think we should be intentional about what information has an actual bearing on their use of the data and what is just implementation detail.

I'd propose:

> pseudopeople is a Python package that generates realistic simulated data about a
fictional United States population, for use in testing entity resolution methods or other data science algorithms. The fictional population was generated by stochastically simulating multiple decades of population dynamics such as fertility, mortality, migration and employment. pseudopeople builds on this population by simulating errors in the data collection process to create realistic, noisy datasets.

But, feel free to disregard all of this!